templates/repo/issue/labels/label_list.tmpl

```diff
@@ -1,6 +1,14 @@
-<h4 class="ui top attached header">
-	{{.locale.Tr "repo.issues.label_count" .NumLabels}}
-	<div class="ui right">
+<h4 class="ui top attached header gt-df">
+	<span class="gt-f1">{{.locale.Tr "repo.issues.label_count" .NumLabels}}</span>
+	<label class="gt-f1" data-tooltip-content={{.locale.Tr "repo.issues.label_archive_tooltip"}}>
+		<input type="checkbox"
+			name="show_archived_labels"
+			class="show-archived-labels-checkbox"
+		>
+		{{.locale.Tr "repo.issues.show_archived_labels"}}
+	</label>
```

Comment:
It seems that Gitea is not ready for using native checkbox at the moment, due to the misalignment

![image](https://github.com/go-gitea/gitea/assets/2114189/0943f7d2-6617-4ee0-b457-8c23741d9ea5)

Reply:
Done

Reply:
<img width="1343" alt="image" src="https://github.com/go-gitea/gitea/assets/80308335/056f2d2d-016c-4098-ad1c-d474a512c6f8">

Reply:
![image](https://github.com/go-gitea/gitea/assets/80308335/9076078e-7d6f-45a2-9479-7dad64f92252)